Yoshi.podspec

```diff
@@ -7,26 +7,18 @@
 #
 
 Pod::Spec.new do |s|
-  s.name             = "Yoshi"
-  s.version          = "3.0.0"
-  s.summary          = "A helpful companion for your iOS app."
-
-# This description is used to generate tags and improve search results.
-#   * Think: What does it do? Why did you write it? What is the focus?
-#   * Try to keep it short, snappy and to the point.
-#   * Write the description between the DESC delimiters below.
-#   * Finally, don't worry about the indent, CocoaPods strips it!
-  s.description      = <<-DESC
-		       Yoshi is a convenient wrapper around the UI code that is often needed for displaying debug menus. Out of the box, Yoshi provides easy-to-implement date, list and custom menus.
-                       DESC
-
-  s.homepage         = "https://github.com/prolificinteractive/Yoshi"
-  s.screenshots     = "https://raw.githubusercontent.com/prolificinteractive/Yoshi/a6e85e87cbd67f2bb3bfe60157e7b13281d80f20/Images/Yoshi.png", "https://raw.githubusercontent.com/prolificinteractive/Yoshi/c66cdf8dc2ab643fe57996d20d3cd37b8b70ceff/Images/Yoshi_iPad.png"
-  s.license          = { :type => 'MIT', :file => 'LICENSE' }
-  s.author           = { "Michael Campbell" => "Michael@prolificinteractive.com" , "Luna An" => "luna@prolificinteractive.com"}
-  s.source           = { :git => "https://github.com/prolificinteractive/Yoshi.git", :tag => s.version.to_s }
-
-  s.platform     = :ios, '8.0'
+  s.name             = 'Yoshi'
+  s.version          = '3.0.1'
+  s.summary          = 'A helpful companion for your iOS app.'
+  s.license          = 'MIT'
+  s.homepage         = 'https://github.com/prolificinteractive/Yoshi'
+  s.authors          = { 'Michael Campbell' => 'Michael@prolificinteractive.com' , 'Luna An' => 'luna@prolificinteractive.com'}
+  s.source           = { :git => "https://github.com/prolificinteractive/Yoshi.git", :tag => s.version }
+  s.screenshots      = 'https://raw.githubusercontent.com/prolificinteractive/Yoshi/a6e85e87cbd67f2bb3bfe60157e7b13281d80f20/Images/Yoshi.png", "https://raw.githubusercontent.com/prolificinteractive/Yoshi/c66cdf8dc2ab643fe57996d20d3cd37b8b70ceff/Images/Yoshi_iPad.png'
+
+  s.ios.deployment_target = '8.0'
+  s.swift_versions   = ['4.2', '5.0', '5.1']
```

Comment:
There is an open PR that takes care of Swift 5 changes. Don't we need to merge it before adding 5.x to the list?
See https://github.com/prolificinteractive/Yoshi/pull/83

Reply:
In this PR I also migrated the code to Swift 5 (the other PR wasn't approved yet so I thought about killing two birds with the same stone)

Reply:
I didn't look at the details in the other PR but looked like there was more going on. Are you saying the other PR can be declined all together once this is merged? I guess I will diff it to see what else was in there once we merge but happy to hear your thoughts